### PR TITLE
Created a doc page for the SQLite data integration.

### LINKED ISSUE #5080: [Docs] Create a doc page for the SQLite data integration
## Instructions :page_facing_up:

Please find the detailed instructions on how to create a doc page for the SQLite data integration here: https://docs.mindsdb.com/contribute/data-integration-page

Make sure to check out the [README.md file](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/sqlite_handler/README.md).

## Hackathon Issue :loudspeaker:

MindsDB has organized a hackathon to let in more contributors to the in-database ML world.

Each hackathon issue is worth a certain amount of credits that will bring you prizes by the end of the MindsDB Hackathon.

Check out the [MindsDB Hackathon rules](https://mindsdb.com/mindsdb-hackathon)!

## The https://github.com/mindsdb/mindsdb/labels/first-timers-only Label

We are happy to welcome you on board! Please take a look at the rules below for first-time contributors.

1. You can solve only one issue labeled as https://github.com/mindsdb/mindsdb/labels/first-timers-only. After that, please look at other issues labeled as https://github.com/mindsdb/mindsdb/labels/good%20first%20issue, https://github.com/mindsdb/mindsdb/labels/help%20wanted, or https://github.com/mindsdb/mindsdb/labels/integration.

2. After you create your first PR in the MindsDB repository, please sign our CLA to become a MindsDB contributor. You can do that by leaving a comment that contains the following: `I have read the CLA Document and I hereby sign the CLA`

Thank you for contributing to MindsDB!

### PR DESCRIPTION
**Pull request by sergiloupa**
i have created a doc file that explaines what SQLite is, how to implement it and how to use it!
fixed #5080

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
@sergiloupa 
Please mention the issue number in the PR description: https://github.com/mindsdb/mindsdb/issues/5080

Please sign our CLA so we can merge your PR.

**Comment by sergiloupa** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/data-integrations/SQLite.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+This handler was implemented using the standard sqlite3 library that comes with Python.
+
+The only required argument to establish a connection is db_file. This points to the database file that the connection is to be made to.
+
+Optionally, this may also be set to :memory:, which will create an in-memory database.
+
+##Usage
+In order to make use of this handler and connect to a SQLite database in MindsDB, the following syntax can be used,
+
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/data-integrations/SQLite.mdx:22]: Please correct the code blocks to be like this:

> \```sql
> SQL GOES HERE
> \```

File: docs/data-integrations/SQLite.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+CREATE DATABASE sqlite_datasource
+WITH
+engine='sqlite',
+parameters={
+    "db_file":"example.db"
+};
+~~~~
+Now, you can use this established connection to query your database as follows,
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/data-integrations/SQLite.mdx:31]: Please correct the code blocks to be like this:

> \```sql
> SQL GOES HERE
> \```

**Pull request merged**